pnpm-lock.yaml

```diff
@@ -14934,6 +14611,60 @@ snapshots:
 
   '@opentelemetry/api@1.9.0': {}
 
+  '@opentelemetry/auto-instrumentations-node@0.53.0(@opentelemetry/api@1.7.0)':
```

Comment:
I'm not totally sure why a bunch of opentelemetry packages were added to the lockfile. All I did was remove the `@changesets/cli` dev dependency and run `pnpm i`.